common/functions/tests/it/scalars/maths/log.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::f64::consts::E;
+
+use common_datavalues::prelude::*;
+use common_exception::Result;
+use common_functions::scalars::*;
+
+#[test]
+fn test_log_function() -> Result<()> {
+    #[allow(dead_code)]
```

Comment:
I think this line `#[allow(dead_code)]` can remove

Reply:
thank you for reminding, I've removed it, it's copy-pasted >.<